Recipes/Recipes.API/Program.cs

```diff
@@ -13,6 +14,7 @@
     options.UseNpgsql(builder.Configuration.GetConnectionString("Default")));
 
 builder.Services.AddScoped(typeof(IGenericRepository<>), typeof(GenericRepository<>));
+builder.Services.AddScoped<IGenericRepository<RecipeEntity>, RecipesRepository>();
```

Comment:
This line is probably not needed because of the `builder.Services.AddScoped(typeof(IGenericRepository<>), typeof(GenericRepository<>));` above

Reply:
removed